modules/social_features/social_group/modules/social_group_default_route/social_group_default_route.api.php

```diff
@@ -6,15 +6,35 @@
  */
 
 /**
- * @addtogroup hooks
- * @{
+ * Provide group bundles for which entities redirection will be applicable.
+ *
+ * @return array
+ *   An associative array of group bundles that shows if group tab management
+ *    should be applied and the default routes for applicable group bundle,
+ *    keyed by group bundle.
+ */
+function hook_social_group_default_route_types(): array {
```

Comment:
@ribel The first hook defines which group types should have tab management enabled and what the default routes should be. The second hook provides the ability to override these settings through other modules. For example, if one instance needs the default page for members of a flexible group to be "Topics" instead of "Stream," we can create a separate module with this hook to override the default routes for that specific flexible group type. The main idea is flexibility—allowing customization of default behavior based on instance-specific needs.